src/battle/battle.cpp

```diff
@@ -62,7 +62,7 @@ void Battle(MainCharacter& main_character, std::vector<Creature>& foes)
 		std::cout << "=========================\n";
 		for (const auto& combatant : combatants)
 		{
-			std::cout << combatant.first->get_creature_type() << ": Initiative "
+			std::cout << combatant.first->get_display_name() << ": Initiative "
```

Comment:
We need a way to capitalize the first letter here. I would suggest to create a utils function for it.